cpp/src/Ice/SSL/SchannelEngine.cpp

```diff
@@ -50,6 +50,262 @@ namespace
 {
     mutex globalMutex;
 
+    string errorStatusToString(DWORD errorStatus)
```

Comment:
This was previously a member method, but It was only used in this file. I moved it to a function.